src/main/java/StringCalculator.java

```diff
@@ -0,0 +1,19 @@
+//문자열 계산 클래스.
+public class
+StringCalculator {
```

Comment:
```suggestion
public class StringCalculator {
```

클래스를 책임에 따라 분리하시고, 이름 또한 적절하게 지어주셨군요! 👍
다만 줄바꿈이 잘못된 것 같아요 😂
스포일러긴 하지만, 다음 미션부터는 코드 스타일에 대해서도 신경을 쓰셔야 합니다!

Reply:
넵! 그렇네요! 😮
코드 스타일...알겠습니다!! 💪